Tests/XMTPTests/ClientTests.swift

```diff
@@ -514,4 +521,28 @@ class ClientTests: XCTestCase {
 		let newState = try await alixClient3.inboxState(refreshFromNetwork: true)
 		XCTAssertEqual(newState.installations.count, 1)
 	}
+	
+	func testCreatesASCWClient() async throws {
+		throw XCTSkip("TODO: Need to write a SCW local deploy with anvil")
```

Comment:
I spent a bunch of time trying to get a SCW running locally like I did in Android. But I'm just not as familiar with the tooling in iOS and I time boxed it. Will revisit when I have more time. Will use the example in RN to validate.